### ISSUE TITLE
system: compact services widget

### ISSUE THREAD

**Issue by fichtner** [opnsense/core#7887]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Is your feature request related to a problem? Please describe.**

The service widget could avoid wasted empty space. Sometimes the list is too long and you need to scroll down inside the widget.

**Describe the solution you like**

Make a more compact tile-based service widget.

**Describe alternatives you considered**

N/A

**Additional context**

N/A

**Comment by sopex** [comment]
Something like these? @fichtner 

<img width="509" height="286" alt="Image" src="https://github.com/user-attachments/assets/73a579c4-9710-4718-8781-3679ba9c17d2" />

<img width="516" height="287" alt="Image" src="https://github.com/user-attachments/assets/8d5c809b-2837-4683-9af5-9b8e91e5ecaa" />

**Comment by fichtner** [comment]
Yes, nice, that was basically what I was looking for. We can condense more space I think but the direction is correct. Want to do a PR?
